src/compiler/diagnosticMessages.json

```diff
@@ -6433,6 +6433,11 @@
         "category": "Message",
         "code": 90053
     },
+    "Includes imports of types referenced by '{0}'": {

```

Comment:
Note that this is never surfaced by the VS Code UI 🤷

I think that's totally fine, I only made it because the protocol made me have a message to go along with the fix